src/view.js

```diff
@@ -216,7 +216,7 @@ export class View {
     let end = this.lastChild;
     let next;
 
-    while (true) {
+    while (true && current) {
```

Comment:
I think on this line we would only need the `while(current)` check. We can remove the `true`. If you can make that change, I'll get this in.